development/forms.py

```diff
@@ -2,33 +2,49 @@
 import requests
 from environment import get_env_variable
 
+from auth_app.models import Bot
+
 
 SERVER_URL = get_env_variable('SERVER_URL')
 SERVER_PORT = get_env_variable('SERVER_PORT')
 
-my_bots = [
-    ('0', 'Pedro'),
-]
-
 
 class ChallengeForm(forms.Form):
     def __init__(self, *args, **kwargs):
+        user = kwargs.pop('user')
         super().__init__(*args, **kwargs)
+        users = get_data_users()
+        online_bots = get_online_bots(users)
+        my_bots = get_my_bots(
+            user,
+            users,
+        )
         self.fields['bot1'].choices = my_bots
-        self.fields['bot2'].choices = get_online_bots()
+        self.fields['bot2'].choices = online_bots
 
     bot1 = forms.ChoiceField(label='MyBots', widget=forms.Select, choices=[])
     bot2 = forms.ChoiceField(label='Online Bots', widget=forms.Select, choices=[])
 
 
-def get_online_bots():
+def get_data_users():
     try:
         bots_json = requests.get(
             '{}:{}/users'.format(SERVER_URL, SERVER_PORT),
         )
         data = bots_json.json()
-        on_bots = [(str(i), bot) for i, bot in enumerate(data['users'])]
-        return on_bots
+        users = data['users']
     except Exception:
-        on_bots = []
-        return on_bots
+        users = []
+
+    return users
+
+
+def get_online_bots(users):
+    return enumerate(users)
+
+
+def get_my_bots(user_online, online_bots):
```

Comment:
Should these methods live on `forms.py`? IMO, looks like helper/domain methods..

Reply:
done!